engine/src/parameters.rs

```diff
@@ -219,10 +221,6 @@ impl TryFrom<NEP141FtOnTransferArgs> for String {
     type Error = ParseError;
 
     fn try_from(value: NEP141FtOnTransferArgs) -> Result<Self, Self::Error> {
-        if !is_valid_account_id(value.sender_id.as_bytes()) {
```

Comment:
With this error condition removed(*), this conversion looks infallible, so should be `From` instead of `TryFrom`.

(*) And I agree it should be removed, the validation makes much more sense in the `TryFrom<JsonValue>` implementation, nice job there.

Reply:
I agree. It was changed.